docs/email/templates/code/promotional.html

```diff
@@ -37,15 +37,11 @@
             mso-table-rspace: 0pt !important;
         }
 
-        /* What it does: Fixes webkit padding issue. Fix for Yahoo mail table alignment bug. Applies table-layout to the first 2 tables then removes for anything nested deeper. */
+        /* What it does: Fixes webkit padding issue. */
```

Comment:
Probably out of scope for this PR but since you're there... Can we ditch "What it does" string – it's used very often here and I'm not sure if it's required to understand the comment.

Reply:
We can remove it if you think it's superfluous.

Reply:
I actually enjoy the what it does